Pennsieve/pennsieve-app
# updated front/login page

# Description
Updated login/entry page for the platform:


# How Has This Been Tested?
Locally:
1. Check login works
2. Check links to public datasets work
3. Check that search works
4. Check all  links.


# Checklist:
